config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,117 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Should some of these defaults be turned on, as with: https://github.com/alphagov/local-links-manager/blob/6ad0ffedb026ec3be91038ebd4e467223c9528a2/config/initializers/new_framework_defaults.rb ?

Reply:
Ah, yes - good point!  I'll work through them. Thanks

Reply:
Gone though them all and updated `config.load_defaults` - all looks good. But, I had to uncomment `require "active_storage/engine"` as there are some active_storage options and I didn't seem to be able to get around without enabling it.

Reply:
Could you add something to the commit message about how you found this out?